.gitignore

```diff
@@ -7,6 +7,7 @@ build
 dist
 *~
 ????-*.patch
+.env
```

Comment:
We use .env files in some of the legacy unit tests on the agent side, perhaps make this for just the server side?

Reply:
E.g.: https://github.com/distributed-system-analysis/pbench/blob/main/agent/bench-scripts/tests/pbench-trafficgen/test-39.env

Reply:
I think you are right so ignore more comment.